code/_core/obj/item/weapon/ranged/bullet/magazine/rifle/pkm.dm

```diff
@@ -2,17 +2,22 @@
 	name = "\improper 7.62x54mmR PKM"
 	desc = "Killa's weapon of choice."
 	desc_extended = ""
-	icon = 'icons/obj/item/weapons/ranged/rifle/rev/762_lmg.dmi'
+	icon = 'icons/obj/item/weapons/ranged/rifle/PKM_old.dmi'
```

Comment:
Why was this changed?

Reply:
to be honest i'm quite new at using github so that shouldn't have been included in this PR and was in the PKM fix PR #429  ,  as the 762_lmg.dmi is missing a sprite for the loaded PKM so the gun just disappears when loaded, this resolves that by switching it back to PKM_old.dmi which does have that sprite until a spriter is able to fix the new sprites.

i'm not quite sure how to remove that from this PR so i'll just close this one out and re-do the description fixes